configs/albu_example/mask-rcnn_r50-fpn_albu-1x_coco.py

```diff
@@ -0,0 +1,68 @@
+_base_ = '../mask_rcnn/mask-rcnn_r50-fpn_1x_coco.py'
```

Comment:
does mask_rcnn also need to rename to `mask-rcnn`

Reply:
Folders do not need to be renamed.